cmd/jujud/reboot/reboot.go

```diff
@@ -21,6 +23,8 @@ import (
 	"github.com/juju/juju/container"
 	"github.com/juju/juju/container/factory"
 	"github.com/juju/juju/instance"
+	"github.com/juju/juju/service"
+	"github.com/juju/juju/service/common"
 )
 
 var logger = loggo.GetLogger("juju.cmd.jujud.reboot")
```

Comment:
@reedobrien This is not the only place where a logger instance is used like this. As @howbazaar  said, if you take a look at the  juju-core source code, its used pretty often.